index.php

```diff
@@ -1,18 +1,8 @@
-<!--Created by Akanksha
-    Desc: First page when app opened, displays logo and name of org and redirects to login after few seconds-->
-<!DOCTYPE html>
-<html>
-<head>
-    <title>FirstAide</title>
-    <link rel="stylesheet" type="text/css" href="css files/index-style.css"/>
-</head>
-<body>
-<center>
-    <img src="images/PClogoWhite.png" style="width:200px; height:200px;float:left;"/>
-    <h1 class="title">First Aide</h1>
-</center>
-</body>
 <?php
-    header("refresh:2;url=login.php");//redirect to login automatically after refreshing 2 times
-?>
-</html>
\ No newline at end of file
+	$APPLICATION_DIR = dirname(__FILE__);
+	require_once($APPLICATION_DIR.'/includes/page_top.php');
+	require_once($APPLICATION_DIR.'/includes/menu.php');
```

Comment:
@AkankshaBodhankar Yes, I have already added the splash screen. It will be reflected in the next PR.